core/src/main/scala/com/ironcorelabs/recrypt/internal/Key.scala

```diff
@@ -37,9 +37,10 @@ final case class PublicKeyGen[FpType <: BigInt](generator: HomogeneousPoint[FpTy
 
 //Maybe this should be arbitrary bytes, but it should be functionally identical.
 final case class PrivateKey[FpType <: BigInt](fp: FpType) extends AnyVal {
-  // def toByteVector: ByteVector = ByteVector(toBigInt.toByteArray)
+  def toByteVector: ByteVector = ByteVector(toBigInt.toByteArray)
```

Comment:
This doesn't guarantee that the private key is padded to the appropriate length so we should remove it and instead do the conversion from the BigInt to ByteVector in CoreApi.scala by calling Fp.bigIntToByteVector instead there.

Do me a favor and delete this instead of recommenting it so we don't make this mistake in the future.

Reply:
Done